app/src/main/java/com/demoapp/App.kt

```diff
@@ -17,8 +18,7 @@ class App : Application() {
             .build()
     }
 
-    fun getAppComponent(): AppComponent {
+    override fun getAppComponent(): AppComponent {
         return appComponent
     }
-
 }
```

Comment:
end line. Change this in your Android Studio settings to ensure end line in all files.